HOW/social/process/how/how-v1.md

```diff
@@ -0,0 +1,164 @@
+---
+HOW: /social/process/how
+Source: /social/process/how/how-v1.md
+Status: Draft
+Created: 2024-11-28
+Version: idx/1
+---
+
+## Title
+HOW Default Process
+
+## Author(s)
+
+zippy <eric@holochain.org>, mattyg <matt@buildyourweb.app>
+
+## Abstract
+
+This HOW describes a default process, format and workflow for how HOWs come into being. HOWs are the primary mechanism for proposing new features, collecting community input on an issue, and documenting decisions affecting Holochain and/or standards and processes used by Holochain hApps or by the Holochain ecosystem.  In other words, this HOW describes **how** the community comes into alignment.
+
+## Motivation
+
+As Holochain and the Holochain ecosystems evolves, we want a standardized process for proposing, adopting, and documenting standards of all types, including: protocol creation and changes, feature additions, and process improvements, etc. A well-defined HOW process helps:
+- Maintain transparency in decision-making
+- Ensure proper technical documentation
+- Foster community participation
```

Comment:
This concerns me. We really don't have the tooling in place in Holochain to make this a resilient space to host content. If somebody decides to engage negatively or just flood in comments then we have limited options for dealing with it.

I suggest that we need to create a separate document for discussion that describes how we operate How. Not a proposal but just a technical guide for ourselves. Like what we do with always-online nodes and backups.

Reply:
The idea is that HOW is a github-only process for now. When we and holochain are ready it can be migrated to a happ.